Cargo.toml

```diff
@@ -18,6 +18,7 @@ default-members = ["wgpu", "wgpu-hal", "wgpu-info"]
 
 [patch."https://github.com/gfx-rs/naga"]
 #naga = { path = "../naga" }
+naga = { git = "https://github.com/JCapucho/naga", branch = "glsl-out-push-constants-v2" }
```

Comment:
This is using a patched `naga` for now to allow for iterating, I'll merge it upstream after we converge on a desing and update `naga`